pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -35,7 +37,46 @@
   {
    "cell_type": "code",
    "execution_count": 2,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Successfully loaded idaes.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "# If running on Google Colab, install Ipopt via IDAES\n",
+    "if \"google.colab\" in sys.modules:\n",
+    "    !wget \"https://raw.githubusercontent.com/ndcbe/CBE60499/main/notebooks/helper.py\"\n",
```

Comment:
Is this the best place to distribute helper.py?  Maybe it should be incorporated into the idaes-pse repo? @adowling2?

Reply:
@hglynch you should change the intro code to:
```
if "google.colab" in sys.modules:
    !wget "https://raw.githubusercontent.com/idaes-pse/main/scripts"
    import colab_helper
    colab_helper.install_idaes()
    colab_helper.install_ipopt()
```